sql/2019/12_Mobile_Web/12_04.sql

```diff
@@ -0,0 +1,34 @@
+#standardSQL
+
+# <meta viewport> not scaleble
```

Comment:
This one should query the `meta-viewport` Lighthouse audit and calculate the % that fail.